app/(sign-up)/step-context.js

```
@@ -0,0 +1,121 @@
+    userInfo: {
+      username: '',
+      birth: '',
+      gender: '',
+    },
+    channelInfo: [],
+    tempChannel: {
+      platform: '',
+      channelId: '',
+      interests: [],
+    },
+  });
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)